src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -56,7 +55,7 @@ public LineResponse updateLine(LineRequest lineRequest, Long id) {
 
     public LineResponse addSection(Long id, SectionRequest sectionRequest) {
         Line line = findLineById(id);
-        addSectionByRequest(line, sectionRequest.getUpStationId(), sectionRequest.getDownStationId(), sectionRequest.getDistance());
+        addSectionBySectionInfo(line, sectionRequest.getUpStationId(), sectionRequest.getDownStationId(), sectionRequest.getDistance());
```

Comment:
```suggestion
        addSection(line, sectionRequest.getUpStationId(), sectionRequest.getDownStationId(), sectionRequest.getDistance());
```

메서드명 뿐만 아니라 파라미터도 충분히 의미를 전달할 수 있지 않을까요?